### ISSUE TITLE
Ethos Suite 1.2.7 for Mac missing. Why only for Windows?

### ISSUE THREAD

**Issue by BernhardFe** [FrSkyRC/ETHOS-Feedback-Community#1670]
There is an Ethos suite for Windows but not for Mac. Will Macs be not supported any more? I have no Windows client and was happy that Macs have been supported in the past for Ethos. But why ist is not provided any more in the last versions (1.2.6 and 1.2.7)? BR Bernhard

**Comment by ZeekWorld** [comment]
Agree, I have installed Ethos Suite Version 1.2.1 on Monterrey. Not sure why the latest versions do not include the mac install.
1.2.1 has a small bug, can't quit the program if I don't force it to shut down.

**Comment by BernhardFe** [comment]
That's true. In 1.2.1. the small bug is existing. I have it too. 
Additionally a small hint. The size of the logo is too big compared  to other logos in the task bar. Modern logos are smaller.
<img width="314" alt="Bildschirmfoto 2022-05-18 um 07 32 39" src="https://user-images.githubusercontent.com/86232280/168964432-f6ff686b-ca34-46dd-a5e8-6787d3b51ac4.png">.

**Comment by BernhardFe** [comment]
Hi Bertrand,
Some comments to the project management for ETHOS. The job you are doing with you team is great and I love the ETHOS. But there is always some room for improvement. So some hints from my side.
Why is this ticket closed if the solution is not available? In Agile IT projects normally a ticket would go into the backlog and will be labeled "In work" until the issue is solved. It will be assigned to a future release plan to make transparent in which sprint it will be solved. Closed is normally, if the customer did not answer after solution within a time frame automatically or the solution ist presented. If customers are not satisfied a ticket can be reopened by them. So to show the status "closed" is in this case not the right one. 
I'm in IT and working with Agile projects. So it would be a great benefit for team members and contributors to have a better overview and produce less questions to the programmer.
Maybe this project can be organised a little bit more transparent? BR Bernhard

**Comment by bsongis-frsky** [comment]
> Why is this ticket closed if the solution is not available?

Because you closed yourself the issue

**Comment by BernhardFe** [comment]
Very sorry my fault :-(
